packages/map-toolkit/src/map-mode/store.test.ts

```diff
@@ -1,749 +0,0 @@
-/*
```

Comment:
these were testing implementation details of the class and the integration tests in use-map-mode test do a better job of checking actual user interaction stuff